localstack-core/localstack/aws/app.py

```diff
@@ -45,6 +45,7 @@ def __init__(self, service_manager: ServiceManager = None) -> None:
                 handlers.add_region_from_header,
                 handlers.rewrite_region,
                 handlers.add_account_id,
+                handlers.parse_trace_context,
```

Comment:
I think this is good! At first, I thought it'd be nicer to have it before the edge routes, because we could also make use of this in API Gateway `execute-api` routes for example. However, we do not pass the `RequestContext` to the edge routes at the moment. (@thrau laments this design decision still today 😄) so there is no point in making it happen early for now. We can always move it later on if anything changes! But for now, this looks to me 👌

Reply:
I see. Feel free to change later :)